packages/manager/modules/pci/src/components/project/flavors-list/flavors-list.constants.js

```diff
@@ -3,22 +3,27 @@ export const DEFAULT_OS = 'linux';
 export const CATEGORIES = [
   {
     category: 'balanced',
+    title: 'General Purpose',
```

Comment:
So the text will be `General Purpose` for all languages ?

Reply:
Yes.